addons/sys_data/XEH_postInitServer.sqf

```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 
-NO_CLIENT;
+if (!isServer) exitWith {};
```

Comment:
This is redundant, this postInit is already called via `serverInit` in XEH.

Mind, that is not the same for `clientInit` combined with `!hasInterface` due to headless clients which are clients without an interface and most of the time don't need either. However `serverInit` excludes headless clients as well.